### ISSUE TITLE
Icon Request: fa-dogs

### ISSUE THREAD

**Issue by sathyabharathy** [FortAwesome/Font-Awesome#6753]
Hello Team,
Before starting to issue my request, I should thank this FontAwesome team for this wonderful work. keep going geeks! These icons are playing a vital role in all of my sites. 
I just need a help from you guys, Can you develop me a dog icon? It would be great help for me!
Thanks in Advance :+1:  :)

**Comment by tagliala** [comment]
thanks!

dog is a duplicate of #1317 , please +1 that request

if you need this icon in a short while, please consider a custom build: https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome

closing here
